sklearn/datasets/_samples_generator.py

```diff
@@ -549,7 +549,8 @@ def make_regression(n_samples=100, n_features=100, *, n_informative=10,
     y : array of shape [n_samples] or [n_samples, n_targets]
         The output values.
 
-    coef : array of shape [n_features] or [n_features, n_targets], optional
+    coef : array of shape (n_features,) or (n_features, n_targets), \
+        default=False
```

Comment:
This is a return type, it does not have a default return. The docstring starts that it is only returned when `coef_` is True.

```suggestion
    coef : array of shape (n_features,) or (n_features, n_targets)
```